### PR TITLE
TS: Clean up.

### PR DESCRIPTION
**Pull request by linbingquan**
It is a little hard to found...

### CODE REVIEW

**Comment by Mugen87** [comment]
@mrdoob Should deprecated method or function parameters always be removed from the TS declaration? I'm not sure this was ever discussed. This PR removes `heightScale` from `TorusKnotBufferGeometry` (which is totally correct) but it could also remove it from `TorusKnotGeometry`.

**Comment by mrdoob** [comment]
Yes, I think d.ts files should not have deprecated stuff.

**Comment by Mugen87** [comment]
@linbingquan Okay, in this case `heightScale` can be removed from `TorusKnotGeometry`, too.

**Comment by linbingquan** [comment]
> @linbingquan Okay, in this case `heightScale` can be removed from `TorusKnotGeometry`, too.

OK.

How about it `TubeGeometry#taper`.

https://github.com/mrdoob/three.js/blob/a24d211f16e961aecb41f80ebb3127fd29ba24fc/src/geometries/TubeGeometry.js#L33

File: src/geometries/TorusKnotGeometry.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { Vector3 } from '../math/Vector3.js';
 
 // TorusKnotGeometry
 
-function TorusKnotGeometry( radius, tube, tubularSegments, radialSegments, p, q, heightScale ) {
+function TorusKnotGeometry( radius, tube, tubularSegments, radialSegments, p, q ) {
```
Comment (Mugen87) [inline src/geometries/TorusKnotGeometry.js:15]: The idea was to remove `heightScale` only from the TS file.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**